shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/ClusterContextManagerCoordinatorTest.java

```diff
@@ -250,4 +258,63 @@ private OptimizerContext createOptimizerContext() {
         when(result.getFederationMetaData().getSchemas()).thenReturn(schemas);
         return result;
     }
+    
+    @Test
+    public void assertRenewPrimaryDSNames() {
+        PrimaryStateChangedEvent mockPrimaryStateChangedEvent = mock(PrimaryStateChangedEvent.class);
+        QualifiedSchema qualifiedSchema = mock(QualifiedSchema.class);
```

Comment:
Is it necessary to mock `QualifiedSchema`? If the object is easy to create, please consider about using `new` at first priority.

Reply:
+1. I'll update the code